src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,49 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+/**
+ * A command that stores the command to find all tasks that match the specific term as a substring. The action of
+ * finding the tasks that match can be carried out when called.
+ */
+public class FindCommand extends Command {
+    /**
+     * The finding term.
+     */
+    private final String TERM;
```

Comment:
Perhaps can name it "KEYWORD" instead of "TERM".

Reply:
Good suggestion. Thanks!